create-proxy.sh

```diff
@@ -30,11 +33,17 @@ fi
 echo "Creating container [${NAME}] using image [${CONTAINER_IMAGE}:${CONTAINER_VERSION}]."
 
 if [ -e "${PSK_FILE}" ]; then
-  COMMAND="chown zabbix:zabbix \"/var/lib/${PSK_FILE}\"; chmod 600 \"/var/lib/${PSK_FILE}\"; ${START_CMD}"
-else
-  COMMAND="${START_CMD}"
+  COMMAND="chown zabbix:zabbix \"/var/lib/${PSK_FILE}\"; chmod 600 \"/var/lib/${PSK_FILE}\";"
+if [ -e "${CA_FILE}" ]; then
+  COMMAND="chown zabbix:zabbix \"/var/lib/${CA_FILE}\"; chmod 600 \"/var/lib/${CA_FILE}\";"
```

Comment:
Also, these commands now lack START_CMD from the end. Maybe you have meant to concatenate COMMAND to the previous one? Like COMMAND="${COMMAND} chown..."